src/pages/index.js

```diff
@@ -0,0 +1,327 @@
+import "./index.css";
+
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+  disableButton,
+} from "../scripts/validation.js";
+
+import { setButtonText } from "../utils/helpers.js";
+import logoSrc from "../images/logo.svg";
+import avatarSrc from "../images/avatar.jpg";
+import editIconSrc from "../images/edit-icon.svg";
+import plusIconSrc from "../images/plus.svg";
+
+import Api from "../utils/Api.js";
+
+document.getElementById("header-icon").src = logoSrc;
+document.querySelector(".profile__avatar").src = avatarSrc;
```

Comment:
`document.querySelector(".profile__avatar")`


You've already found `profileAvatar`. Please, move it into line 17 to be able to use it here


```js

import Api from "../utils/Api.js";

const profileAvatar = document.querySelector(".profile__avatar");

document.getElementById("header-icon").src = logoSrc;
profileAvatar.src = avatarSrc;  // here
document.getElementById("edit-icon").src = editIconSrc;
document.getElementById("plus-icon").src = plusIconSrc;
```